com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolumeProfile.cs

```diff
@@ -18,23 +19,21 @@ public sealed class ProbeReferenceVolumeProfile : ScriptableObject
         /// <summary>
         /// The size of a Cell.
         /// </summary>
+        [Min(1)]
         public int cellSize = 64;
         /// <summary>
         /// The size of a Brick.
         /// </summary>
-        public int brickSize = 4;
-        /// <summary>
-        /// Max subdivision.
-        /// </summary>
-        public int maxSubdivision = 2;
+        [Min(0.25f)]
+        public float minDistanceBetweenProbes = 1.0f;
         /// <summary>
         /// The normal bias to apply during shading.
         /// </summary>
+        [Range(0.0f, 1.0f), Delayed]
         public float normalBias = 0.2f;
-        /// <summary>
-        /// Index field dimensions.
-        /// </summary>
-        public Vector3Int indexDimensions = s_DefaultIndexDimensions;
+
+        public int maxSubdivision => Mathf.CeilToInt(Mathf.Log(cellSize / minDistanceBetweenProbes, 2));
+        public int brickSize => (int)(minDistanceBetweenProbes * 4.0f);
```

Comment:
Note that this is not correct, while it is true we have 4 probes per brick, it means that the distance between probe is BrickSize/3 not 4 (see crude visualization below)  

X ---- X ---- X ---- X

Reply:
Oops, I forgot to change it, sorry 😅